proto/src/ground/v1beta1/submission.proto

```diff
@@ -102,7 +102,7 @@ message TaskData {
   // A manually entered number response.
   message NumberResponse {
     // The number provided by the user.
-    double number = 2;
+    double number = 1;
```

Comment:
This will be a breaking change for any recently serialized surveys, which are all tests, but I think this is more correct.

Reply:
Ack. There's no real data in the db at the moment, and we haven't rolled out this change to prod, so I think this is safe.